packages/field-content/src/views/Controller.js

```diff
@@ -71,6 +67,59 @@ export default class ContentController extends TextController {
     });
   }
 
+  getFilterGraphQL = ({ type, value }) => {
```

Comment:
Instead of extending the `Text` controller, we bring the implementation in directly and modify it slightly so it's more appropriate for the Content type.